### ISSUE TITLE
Show used IP of AGH

### ISSUE THREAD

**Issue by misery** [AdguardTeam/AdGuardHome#5616]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to request a feature or enhancement and not ask a question

#### What problem are you trying to solve?
My AGH has multiple IPs. One IP is the main IP that is distributed by DHCP. Another IP is used to "rewritten" DNS-Requests. My firewall rewrites any outgoing DNS request to udp/53 to the seconds AGH IP.
It would be nice if I could see the used IP in "client details" of that request.

#### Proposed solution
Show used (internal) IP of AGH in client detail (or somewhere else) of a dns request.

**Comment by ainar-g** [comment]
Hello.  I'm not sure I understand the feature request.  All known IP addresses of AdGuard Home are shown on the *Setup Guide* page.  You haven't mentioned in the post, but if AdGuard Home is the DHCP server in this configuration then it sends all IP addresses (of the appropriate protocol) of the configured network interface to the DHCP clients.

**Comment by misery** [comment]
Yes, I already know any IP of AGH in my setup. But I cannot see which IP the client used for the DNS request. This is not necessary for USUAL use-case.

1. Use case: Do not propagate all IPs
My "seconds IP" of AGH is not known by my client. It will be used by my firewall rewrite only. So I could see in AGH which requests was re-written by my firewall because the destination IP of AGH is not the normal IP.

2. Use case: Propagate all IPs by DHCP
So I can see which client used the second entry. Maybe because the first entry (request) was blocked by any other firewall or something like this.

Nevertheless.... would just be nice if AGH could show the used IP of AGH in logs.

**Comment by ainar-g** [comment]
Ah, I see, so you mean showing the local address (in server's terms) in the query log?

**Comment by misery** [comment]
Yes :-)

**Comment by nmpu** [comment]
This-- but also add as a filter for persistent client. For example, primary AdGuard server address includes ad blocking while alternate AdGuard server address omits ad blocking. This would allow a client to manually disable ad blocking without administrative changes. So, under this scheme, client settings would include both client AND server identifier. I currently have to run multiple instances to achieve this functionality.
